mmeds/server.py

```diff
@@ -553,13 +602,19 @@ def __init__(self, watcher, q, testing=False):
     ######################################
 
     @cp.expose
-    def run_analysis(self, access_code, tool_type, analysis_type, config):
+    def run_analysis(self, access_code, tool, config):
         """
         Run analysis on the specified study
         ----------------------------------------
         :access_code: The code that identifies the dataset to run the tool on
         :tool: The tool to run on the chosen dataset
         """
+        if '-' in tool:
+            tool_type = tool.split('-')[0]
+            analysis_type = tool.split('-')[1]
+        else:
+            tool_type = tool
+            analysis_type = 'default'
```

Comment:
I am moving away from the combined properties for tool, so `tool_type` and `analysis_type` should be separate parameters

Reply:
'tool' is the input received from the html file 'analysis_select_tool.html' There is only one drop down on that page, and the response right now is in the format of tool_type-analysis_type (or just tool_type for sparcc and lefse). This bit of code split up the input into the respective variables, so 'tool' is never used after this. 
If it's really important that tool_type and analysis_type be defined from the input directly, I can change the html page to have two drop down menus, one for tool_type and one for analysis_type. However, I thought that the one drop down menu and then splitting the input into two in server.py is more user friendly. I can rename the variable something other than tool, to make it more clear.

Reply:
I think this is okay for now. Looks like I didn't update the html in the last branch I merged to master. Eventually I'd like to have one drop down be the tool and then based on which tool was selected there would pop up a second menu for additional options which would then be written to the config file for that analysis run. I think that's outside the scope of this PR though